src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -78,6 +78,16 @@
                             + "E.p. \"0\" for an unsharded keyspace. "
                             + "Or \"-80\" for the -80 shard of the sharded keyspace.");
 
+    public static final Field GTID = Field.create(VITESS_CONFIG_GROUP_PREFIX + "gtid")
+            .withDisplayName("gtid")
+            .withType(Type.STRING)
+            .withWidth(Width.LONG)
+            .withDefault("")
```

Comment:
I think we can omit the default and just react to the value if it's not `null`?

Reply:
yep, done